packages/venia-concept/__mocks__/braintree-web-drop-in.js

```diff
@@ -0,0 +1,7 @@
+const mockCreate = jest.fn();
+
+const dropin = {
+    create: mockCreate
+};
+
+module.exports = dropin;
```

Comment:
Since this is an external module and we want to mock its methods, I've created a [manual mock](https://jestjs.io/docs/en/manual-mocks#mocking-node-modules) for it.